Website/controls/user.ascx

```diff
@@ -30,10 +41,10 @@
 			<asp:label id="lblUsername" runat="server"></asp:label>
 			<asp:requiredfieldvalidator id="valUsername" runat="server" cssclass="NormalRed" display="Dynamic" errormessage="<br>Username Is Required."
 				controltovalidate="txtUsername" resourcekey="valUsername"></asp:requiredfieldvalidator></td>
-	</tr>
+
```

Comment:
I think there is a missing </tr> here

Reply:
arrg, stripped it...
I thinkg there is a missing `</tr>` here

Reply:
odd thanks for catching that dont understand how anything could of been missing.  It was pretty straight forward operation :)